TombLib/TombLib/LevelData/Compilers/TombEngine/LevelCompilerTombEngine.cs

```diff
@@ -73,7 +74,6 @@ public override CompilerStatistics CompileLevel()
 
             DetectTombEngineVersion(false);
             DetectTombEngineVersion(true);
-
```

Comment:
Please don't remove empty lines here and below, it's done to group code in meaningful parts.